container-host-files/opt/scf/bin/klog.sh

```diff
@@ -21,7 +21,8 @@ if [ "$1" == "-f" ]; then
   FORCE=1
 fi
 
-NS=${1-scf}
+NS=${1-scf} 
```

Comment:
Could you revert this change? (there is an extra space at the end and a line break.

Reply:
Done!